engine/src/main/resources/assets/ui/menu/selectGameScreen.ui

```diff
@@ -62,8 +62,21 @@
                     },
                     "position-bottom": {
                         "target": "TOP",
+                        "widget": "saveGamePath",
+                        "offset": 8
+                    }
+                }
+            },
+            {
```

Comment:
This should be preceded by a "Save game path:" label just for consistency (since you're doing that in `createGameScreen`).